code-challenges.js

```diff
@@ -6,36 +6,76 @@
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
 // Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp1 = 35
-var temp2 = 350
-var temp3 = 212
-
-
+ var temp1 = 35
+ var temp2 = 350
+ var temp3 = 212
+//
+ //declare a function called boilPoint
+  const boilPoint = (temp) => {
+//create if/else statment determining if the temp is below,at, or above boiling point
+   if(temp < temp3){
+      return `${temp1} is less than boiling point `
+  } else if(temp > temp3){
+      return `${temp2} is above boiling point `
+  } else if(temp === temp3){
+      return `${temp3} is at boiling point `
+   }
+ }
```

Comment:
great work! i totally follow your logic here. the only issue i see is that you are hard-coding the variables `temp1`, `temp2`, and `temp3` into your function. as it is, if i put _any_ number below boiling point, the return will always say "35 is less than boiling point", etc. which is probably not what you want. 

i think that it'd be fine to use a variable inside of your function but you might consider that variable to be scoped locally to your function using `let`. ill show you what i mean. this if your exact code with some minor tweaks:

```javascript
// function boilPoint takes in a number (temp) as an argument
const boilPoint = (temp) => {
  // create a variable that represents boiling point
  let boilTemp = 212
  //create if/else statement determining if the temp is below, at, or above boilTemp
  if(temp < boilTemp){
    // use string interpolation so that the user's input is what shows in the sentence
    return `${temp} is less than boiling point `
  } else if(temp > boilTemp){
    return `${temp} is above boiling point `
  } else if(temp === boilTemp){
    return `${temp} is at boiling point `
  }
}
```